lib/patcher.js

```diff
@@ -520,5 +521,12 @@ function findChildReprints(newPath, oldPath, reprints) {
         }
     }
 
+    // Return statements might end up running into ASI issues due to comments
+    // inserted deep within the tree, so reprint them if anything changed
+    // within them.
+    if (ReturnStatement.check(newPath.getNode()) && reprints.length > 0) {
```

Comment:
Although this is safe as written, let's save `reprints.length` before the loop and check if it has increased here, so that we only `return false` if the argument changed (there may be other reprints from elsewhere in the tree).